gcloud/datastore/batch.py

```diff
@@ -21,7 +21,7 @@ class Local(object):
 
 from gcloud.datastore import _implicit_environ
 from gcloud.datastore import helpers
-from gcloud.datastore import datastore_v1_pb2 as datastore_pb
+from gcloud.datastore import _datastore_v1_pb2 as datastore_pb
```

Comment:
Yeah this PR isn't adding it to the auto-doc'd page, just making the module name start with an underscore so it's no longer public.

As for as lower-level access, they can always import a module starting with an underscore, they just don't have a contract with us about whether or not it will break in the future (which is fine if they `pip install gcloud==1.0.0` frozen version).

RE: proto support, right know we have `helpers` -- but only for creating `Key` and `Entity` **from** protobufs -- and `Key.to_protobuf` and then every public method in `Connection` takes and returns protobufs.